File: problems/reverse-String/boilerplate-full/function.rs

  use std::io::{self, BufRead};
  
  ##USER_CODE_HERE##
  
  fn main() {
    let stdin = io::stdin();
    let mut input = stdin.lock().lines().map(|l| l.unwrap());
    let input: String = input.next().unwrap().parse().unwrap();
    let result = reverseString(input);
    println!("{}", result);
  }
      

File: problems/reverse-String/boilerplate-full/function.js

  ##USER_CODE_HERE##
  
  const input1 = require('fs').readFileSync('/dev/stdin', 'utf8').trim().split('\n').join(' ').split(' ');
  const input =input1.shift();
  const result = reverseString(input);
  console.log(result);
      